app/src/main/java/org/simple/clinic/enterotp/EnterOtpScreen.kt

```diff
@@ -186,6 +186,15 @@ class EnterOtpScreen : BaseScreen<
     showError(resources.getString(R.string.otpentry_error_incorrect_otp_attempts_limit_reached, attemptsMade.toString()))
   }
 
+  override fun showFailedAttemptOtpError(attemptsMade: Int, attemptsRemaining: Int) {
+    if (attemptsMade in 1..2) {
+      showError(resources.getString(R.string.otpentry_error_incorrect_otp_attempt))
+    } else if (attemptsMade in 3..4) {
+      showError(resources.getString(R.string.otpentry_error_incorrect_otp_attempts_remaining, attemptsRemaining.toString()))
+    }
```

Comment:
Instead of this, could we have just used this? 

```suggestion
if (attemptsMode >= 3) {
  // show error with attempts remaning
} else {
  // show error
}
```
I think this is much more understandable, wdyt?